x-pack/plugins/infra/public/pages/metrics/hosts/components/search_bar/controls_title.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiFormLabel, EuiPopover, EuiIcon, EuiToolTip, EuiLink } from '@elastic/eui';
+import useToggle from 'react-use/lib/useToggle';
+import { css } from '@emotion/react';
+import { i18n } from '@kbn/i18n';
+import { availableControlsPanels } from '../../hooks/use_control_panels_url_state';
+
+const helpMessages = {
+  [availableControlsPanels.SERVICE_NAME]: {
+    text: `${i18n.translate('xpack.infra.hostsViewPage.serviceNameControl.popoverHelpLabel', {
+      defaultMessage: 'Services detected via',
+    })}`,
+    link: {
+      text: `${i18n.translate('xpack.infra.hostsViewPage.serviceNameControl.popoverHelpLink', {
+        defaultMessage: 'APM',
+      })}`,
+      href: 'https://ela.st/docs-infra-apm',
+      ['data-test-subj']: 'hostsViewServiceNameControlPopoverHelpLink',
+    },
+  },
+};
+
+const TitleWithPopoverMessage = ({
+  title,
+  helpMessage,
+  embeddableId,
+}: {
+  title?: string;
+  helpMessage: {
+    text: string;
+    link?: {
+      href: string;
+      'data-test-subj': string;
+      text: string;
+    };
+  };
```

Comment:
Yes, thanks for spotting that, I will try that ( I made it like that in the previous version when I passed that to the controls because I couldn't pass a component there )

Reply:
Changed ✔️